deeplake/auto/unstructured/coco/convert.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from typing import Dict, Any, Optional
+
+from deeplake.core.tensor import Tensor
+
+
+def coco_to_deeplake(
+    coco_key: str,
+    value: Any,
+    destination_tensor: Tensor,
+    category_lookup: Optional[Dict] = None,
+):
+    """Takes a key-value pair from coco data and converts it to data in Deep Lake compatible format"""
+    dtype = destination_tensor.meta.dtype
+
+    if coco_key == "bbox":
+        assert len(value) == 4
```

Comment:
throw proper exceptions instead of assert wherever needed

Reply:
Converted to expection. Will go through other cases too, but this is the only one I think